src/signal_manager.sh

```diff
@@ -0,0 +1,30 @@
+. "$KW_LIB_DIR/kwio.sh" --source-only
+
+# This functions prints the default message when interrupting kw
+function default_interrupt_handler() {
+  say $'\n'$SEPARATOR
+  say Exiting...
+}
+
+# This function adds a new signal handler to an arbitrary signal
+#
+# @command The handler to be called when a signal is catched
```

Comment:
Not necessarily... it should be a string to be passed to eval. The following is allowed:

`signal_manager 'echo goodbye!' SIGINT SIGTERM`

Reply:
I see, but in this case, your documentation is a little bit wrong because users must use quotes if they want to pass a function with parameters. If you do not use it, you will lose the values.